manifests/mysql_connector.pp

```diff
@@ -0,0 +1,19 @@
+class confluence::mysql_connector (
+  String $version      = $confluence::mysql_connector_version,
+  String $installdir   = $confluence::mysql_connector_install,
+) {
+
+
+  class { 'mysql_java_connector':
+    installdir   => $confluence::mysql_connector_install,
+    version      => $confluence::mysql_connector_version,
+    proxy_server => $confluence::proxy_server,
+  }
+
+  file { "${confluence::installdir}/atlassian-confluence-${confluence::version}/confluence/WEB-INF/lib/mysql-connector-java-5.1.47-bin.jar":
```

Comment:
should this path also use the variable instead of hardcoding the version?

Reply:
my bad :) will fix this of course.